src/modules/keyboard/Config.cpp

```diff
@@ -163,7 +168,14 @@ Config::Config( QObject* parent )
              {
                  // Set Xorg keyboard model
                  m_selectedModel = m_keyboardModelsModel->key( index );
-                 QProcess::execute( "setxkbmap", xkbmap_model_args( m_selectedModel ) );
+                 if ( m_useLocale1 )
```

Comment:
:ship: Since there's multiple places where `if(m_useLocale)` happens, I wonder if there's a more elegant way to do this (and then realise that that would be the enemy of the good, here). Some interface or proxy object to map the work to an underlying implementation.